extensions/amp-bind/amp-bind.md

```diff
@@ -76,48 +141,76 @@ The state can be initialized with the `amp-state` component:
 
 ##### AMP.setState()
 
-State can be mutated by the new `AMP.setState()` [action](../../spec/amp-actions-and-events.md).
+State can be mutated by the [`AMP.setState()` action](../../spec/amp-actions-and-events.md).
 
 - `AMP.setState()` performs a [shallow merge](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) of its arguments with the document state.
 - `AMP.setState()` can override data initialized by `amp-state`.
 
-### Binding
+### Bindings
 
-A binding is a special attribute of the form `[property]` that links an element's property to an [expression](#expressions).
+A **binding** is a special attribute of the form `[property]` that links an element's property to an [expression](#expressions).
 
 When the **state** changes, expressions are re-evaluated and the bound elements' properties are updated with the new expression results.
 
 `amp-bind` supports data bindings on three types of element state:
 
-| Type | Syntax | Details |
+| Type | Attribute | Details |
 | --- | --- | --- |
 | [Node.textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) | `[text]` | Supported on most text elements.
-| [CSS Classes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class) | `[class]` | Expression result must be a space-delimited string.
-| Element-specific attribute | `[<attr>]` | Only whitelisted attributes are supported.<br>Boolean expression results toggle boolean attributes.
+| [CSS classes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class) | `[class]` | Expression result must be a space-delimited string.
+| Size of [AMP elements](https://www.ampproject.org/docs/reference/components) | `[width]`<br>`[height]` | Changes the width and/or height of the AMP element. |
+| Element-specific attribute | [Various](#element-specific-attributes). | Boolean expression results toggle boolean attributes.
 
-- For security reasons, binding to `innerHTML` is disallowed
-- All attribute bindings are sanitized for unsafe URL protocols (e.g. `javascript:`)
+- For security reasons, binding to `innerHTML` is disallowed.
```

Comment:
We prevent binding to a lot of attributes. Do you think this one is worth explicitly calling out?